demos/sdk_output_skeleton_1_ghe/network.geojson

```diff
@@ -705,8 +705,8 @@
         "building_type": "Single-Family Detached",
         "floor_area": 5500,
         "footprint_area": 4655,
-        "number_of_stories_above_ground": 1,
-        "number_of_stories": 2,
+        "number_of_stories_above_ground": 2,
+        "number_of_stories": 3,
```

Comment:
Where did this come from?

Reply:
When I ran OS simulations using this geojson, the simulation failed saying a building with a conditioned attic must have at least 2 stories above ground. Making that change was easy and made this a viable geojson file.